test/e2e/framework.go

```diff
@@ -1410,13 +1410,17 @@ func (data *TestData) waitForClickHousePod() error {
 		// The updating is completed when CurrentRevision is updated to UpdateRevision
 		return ss.Status.CurrentRevision == ss.Status.UpdateRevision, nil
 	})
+	clickHousePodName := fmt.Sprintf("%s-0-0-0", clickHousePodNamePrefix)
```

Comment:
Please use the const mentioned as L172. Could you change at L1323 also?

Reply:
Thanks for reviewing. I've fixed it and also moved clickHouseStatefulSetName to the same place.